boto/gs/connection.py

```diff
@@ -33,14 +34,14 @@ class Location(object):
 class GSConnection(S3Connection):
 
     DefaultHost = 'storage.googleapis.com'
+    DefaultCallingFormat = 'boto.s3.connection.OrdinaryCallingFormat'
     QueryString = 'Signature=%s&Expires=%d&GoogleAccessId=%s'
 
     def __init__(self, gs_access_key_id=None, gs_secret_access_key=None,
                  is_secure=True, port=None, proxy=None, proxy_port=None,
                  proxy_user=None, proxy_pass=None,
                  host=DefaultHost, debug=0, https_connection_factory=None,
-                 calling_format=SubdomainCallingFormat(), path='/',
-                 suppress_consec_slashes=True):
+                 calling_format=None, path='/', suppress_consec_slashes=True):
```

Comment:
Do you want to set the `calling_format` below like you did for `s3/connection.py`?

Reply:
No need since this inherits from the S3 connection class!